lbrynet/daemon/Daemon.py

```diff
@@ -1453,11 +1477,11 @@ def jsonrpc_claim_show(self, txid=None, nout=None, claim_id=None):
                        [<claim_id> | --claim_id=<claim_id>]
 
         Options:
-            <txid>, --txid=<txid>              : look for claim with this txid, nout must
-                                                    also be specified
-            <nout>, --nout=<nout>              : look for claim with this nout, txid must
-                                                    also be specified
-            <claim_id>, --claim_id=<claim_id>  : look for claim with this claim id
+            --txid=<txid>              : (str) look for claim with this txid, nout must
+                                         also be specified
+            --nout=<nout>              : (int) look for claim with this nout, txid must
+                                         also be specified
+            --claim_id=<claim_id>  : (str) look for claim with this claim id
```

Comment:
here too

Reply:
This will be automatically taken care by the scripts in the final output.